syntax = "proto3";

package orderexecutor;

service OrderExecutorService {
}